synapse/app/__init__.py

```diff
@@ -12,3 +12,23 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
+
+import sys
+sys.dont_write_bytecode = True
+
+from synapse.python_dependencies import (
+    check_requirements, MissingRequirementError
+)  # NOQA
+
+if __name__ == '__main__':
```

Comment:
Bleugh, not it doesn't, I checked that the file gets loaded but not the actual functionality.

How about we just remove the __name__ check? Either way this code won't work without its deps...